### DISCUSSION TITLE
udp pull stream！！

### DISCUSSION THREAD

**Discussion by ztaozhang** [bluenviron/mediamtx#1669] [Questions]
![image](https://user-images.githubusercontent.com/47706186/230699300-4e90404a-bd95-46fc-9a50-1b811459da64.png)
![image](https://user-images.githubusercontent.com/47706186/230699383-bf96fa20-84d9-4a89-ac52-beda8fc9f6bd.png)

Cannot support specified network card and ip or vlan to pull stream！

**Comment by aler9** [comment]
Hello, please use the "issues" section to report issues and provide relevant details that allow to replicate the issue.

[a comment by an automated account is not shown]
